### DISCUSSION TITLE
Error with compiling my mod

### DISCUSSION THREAD

**Discussion by StarBrps** [FunkinCrew/Funkin#1974] [Compiling help]
Please help, Whenever I lime test windows for my fnf mod I get this error

"C:/HaxeToolkit/haxe/lib/polymod/git/polymod/fs/PolymodFileSystem.hx:19: characters 16-18 : Expected )"
Did I do something wrong setting it up? (If it helps I'm using KadeEngine 1.5.2)

**Comment by Snirozu** [comment]
don't know how to fix this but maybe try downgrading haxe which i don't suggest you to do or remove all references to polymod in all classes
